### DISCUSSION TITLE
Can_Be_Damaged option for buildable items.

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#2621] [Q&A] [answered]
In update 3.21.18.0, "Can_Be_Damaged" is introduced.
I am assuming it is used for barricades and structure items, placed in the .dat file as a line with true/false beside it.
I am also assuming this works like:
True - The default option, will check for HP, Vulnerable tag and Armor_Tier High tag.
False - Ignores all damage and exists till decayed/salvaged.

Not quite sure though, hence this post to check. Thanks in advance for any replies :)

**Comment by SDGNelson** [comment, marked as answer]
Yes, that is correct.
